### PR TITLE
bumped version to 0.33.0

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -151,4 +151,3 @@ api:
     obr: true
     bom: true
     mvp: true
-    isolated: true
```
Comment (jadecarino) [inline release.yaml:154]: This happened last time for some reason the script cuts off this line which we need at the end - can you add back in please? And look at fixing that error in the script for next time pls (can be tomorrow)?

**Review by jadecarino — CHANGES_REQUESTED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**